packages/babel-parser/test/fixtures/typescript/module-namespace/module-new-line/input.js

```diff
@@ -0,0 +1,2 @@
+module
+Foo = {}
```

Comment:
Since `module Foo = {}` woldn't be valid anyway, maybe the test should be like this?

```js
module
Foo
{}
```

Also for interfaces and namespaces

Reply:
K